authentication/admin.py

```diff
@@ -1,3 +1,7 @@
 from django.contrib import admin
+from authentication.models import User, UserManager
+
+class UserAdmin(admin.ModelAdmin):
+    pass
+admin.site.register(User, UserAdmin)
```

Comment:
PEP8, pls

Reply:
you can use decorator syntax for more readable code